File: python/01.py
#!/usr/bin/env python
import array

input = open('../input_files/01.txt', 'r')

data = []
for line in input:
 datum = line.split()
 data.append(datum)

flattened = [val for sublist in data for val in sublist]
##last = flattened.pop()  ## pop() with no argument grabs the last element.

reverse_list = list(reversed(flattened))
last = reverse_list.pop(0)

print "The final element was: " + str(last)

File: perl/01.pl
#!/usr/bin/env perl
use common::sense;
use autodie;

open(IN, "<../01.txt");
my $input = "";
while (my $line = <IN>) {
    chomp $line;
    $input .= $line;
}
close(IN);
my @list = split(/\s+/, $input);
print "The last element was: $list[$#list]\n";
    
